mvp_example/src/main/java/com/distillery/android/blueprints/mvp/main/contract/TodoContract.kt

```diff
@@ -0,0 +1,17 @@
+package com.distillery.android.blueprints.mvp.main.contract
```

Comment:
Could you describe what `main` means? If it stands for `MainActivity` then it's better to give it a name according to the feature, not the activity name. Also, I think we don't need `contract` folder since it will always contain a single file.
```suggestion
package com.distillery.android.blueprints.mvp.todo
```

Reply:
Yes, kind of refers to that
But I think it would be more descriptive to change the name to `todo` 
Thanks!